plugins/sqlite_authkey_storage_plugin/src/main/kotlin/de/busesteinkamp/plugins/auth/SqliteAuthKeyRepository.kt

```diff
@@ -0,0 +1,77 @@
+package de.busesteinkamp.plugins.auth
+
+import de.busesteinkamp.domain.auth.AuthKey
+import de.busesteinkamp.domain.auth.AuthKeyRepository
+import java.sql.DriverManager
+import java.text.SimpleDateFormat
+import java.util.*
+
+class SqliteAuthKeyRepository : AuthKeyRepository {
+    private val url = "jdbc:sqlite:auth_keys.db"
+
+    init {
+        Class.forName("org.sqlite.JDBC")
+
+        val connection = DriverManager.getConnection(url)
+        val statement = connection.createStatement()
+        statement.executeUpdate(
+            "CREATE TABLE IF NOT EXISTS auth_keys (platformName TEXT, authKey TEXT, createdAt TEXT, expiresAt TEXT)"
+        )
+        statement.close()
+        connection.close()
+    }
```

Comment:
Es kann sein, dass der Driver nicht gefunden wird aus irgendeinem Problem heraus:
```suggestion
    init {
        try {
            Class.forName("org.sqlite.JDBC")

            val connection = DriverManager.getConnection(url)
            val statement = connection.createStatement()
            statement.executeUpdate(
                "CREATE TABLE IF NOT EXISTS auth_keys (platformName TEXT, authKey TEXT, createdAt TEXT, expiresAt TEXT)"
            )
            statement.close()
            connection.close()
        } catch (e: ClassNotFoundException) {
            throw RuntimeException("SQLite JDBC driver not found", e)
        }
    }
```